dts/riscv/bouffalolab/bl6.dtsi

```diff
@@ -24,6 +24,7 @@
 			riscv,pmpregions = <4>;
 
 			ictrl: interrupt-controller {
+				#address-cells = <0>;
```

Comment:
cf5ef26e341822ee74eee7d75621ed62912b06a0 should be squashed

Reply:
I kept separated for historical reasons and would like to keep as is.